bindings/pydeck/README.md

```diff
@@ -117,16 +117,18 @@ python3 setup.py install
 
 ## Development
 
+Build the deck.gl project at the root of this repository:
```

Comment:
Is the documentation change related to the bundle config change?

Reply:
Much of it is–some changes are just for clarity